util/src/main/java/com/liferay/faces/util/application/ComponentResourceUtil.java

```diff
@@ -1,60 +0,0 @@
-/**
- * Copyright (c) 2000-2015 Liferay, Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package com.liferay.faces.util.application;
-
-import javax.faces.component.UIComponent;
-import javax.faces.component.ValueHolder;
-
-
-/**
- * @author  Neil Griffin
- */
-public class ComponentResourceUtil {
-
-	public static String getComponentValue(UIComponent uiComponentResource) {
```

Comment:
The functionality of this method has been moved into [the `private` method `UIViewRootUtilImpl.getComponentValue()`.](https://github.com/liferay/liferay-faces-util/pull/16/files#diff-7f62375232ab1d1df6830cb53c7cc15cR121) I know that `HeadRendererBridgeImpl` uses this method, but it seems inappropriate for it to be public API. The functionality should be duplicated in the bridge if it is still needed after this code is merged.